integrations/analytics/heap.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Heap"
+---
+
+Add the following to your `mint.json` file to send analytics to Heap.
+
+<CodeGroup>
+
+```json Analytics options in mint.json
+"analytics": {
+    "heap": {
+        "appId": "required"
+    }
+}
+```
+
+```json Example
+"analytics": {
+    "heap": {
+        "publicApiKey": "1234567890"
```

Comment:
Isn't it `appId`? Can we also look into what the general format of the appId is like so we can at least replicate that so it's less confusing?

Reply:
it's a ten-digit number, i could make a dummy one!